plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineManifest.kt

```diff
@@ -15,8 +23,14 @@ open class GenerateGramineManifest @Inject constructor(objects: ObjectFactory) :
         const val MANIFEST_TEMPLATE = "java.manifest.template"
     }
 
+    @get:Input
+    val buildType: Property<BuildType> = objects.property(BuildType::class.java)
```

Comment:
Inject

Reply:
What do you mean by "inject"?

Reply:
Use dependency injection. Have a look at the other tasks which take in the build type .

Reply:
Done